apps/server/src/infra/sync/tsp/tsp-sync.strategy.spec.ts

```diff
@@ -286,6 +301,8 @@ describe(TspSyncStrategy.name, () => {
 
 		describe('when school does not exist', () => {
 			const setup = () => {
+				tspSyncService.findTspSystemOrFail.mockResolvedValueOnce(systemFactory.build());
```

Comment:
can also be added to mock service

Reply:
Already in mock services. I think this was caused by the merge of main. I removed the lines